README.md

```diff
@@ -194,6 +194,67 @@ for getting the file list.
 This option cannot be set to `false` with `redirect` set to `true` on a server
 with `ignoreTrailingSlash` set to `true`.
 
+#### `hash`
+
+Default: `undefined`
+
+Enabling `hash` will turn off the `wildcard` option if it wasn't explicitly set to `true`, and both cannot be set to `true`. When enabled, `hash` lets the user access assets dynamically using the decorated `getHashedAsset` function. This in turn makes possible the usage of a very high `maxAge` so that the content can be cached as long as possible. If any modifications are made to a file, its hash will simply be recalculated during the next startup and the cache will bust for that asset.
```

Comment:
Can I suggest `getHashedStaticPath` or something like that that indicates this is from `fastify-static` and that this returns a path, not a whole file or asset? Just reading `getHashedAsset` without reading anything else isn't super explicit about what the return value is, and the rest of the docs don't use the term "asset" much.

Reply:
I agree, this PR essentially started as a fast hack, and molded into the complete version, so names were definitely subject to change